### PR TITLE
solution

### PR DESCRIPTION
**Pull request by vasylmiskiv1**

### CODE REVIEW

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
St review required  ⚠️

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -27,8 +27,16 @@
...
-  // write code here
-}
+const calculateRentalCost = days => {
+  let res = 40 * days;
+
+  if (days >= 7) {
+    res -= 50;
+  } else if (days >= 3) {
+    res -= 20;
+  }
```
Comment (AndreyMalyshev147) [inline src/calculateRentalCost.js:37]:
```suggestion
  };
```

**Review by AndreyMalyshev147 — APPROVED** [review]
Excellent)😃

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -27,8 +27,16 @@
  * @return {number}
  */
 
-function calculateRentalCost(days) {
-  // write code here
-}
+const calculateRentalCost = days => {
+  let res = 40 * days;
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:31]: Let's create more meaningful name for variables:
```suggestion
  let totalCost = 40 * days;
```

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**